task/kind/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Add git-clone
+add_task kind latest
```

Comment:
we don't need to re-create the `kind` task again as the task which needs to be tested is already created.